packages/template-retail-react-app/app/pages/checkout/index.jsx

```diff
@@ -46,6 +47,10 @@ const Checkout = () => {
     const {data: basket} = useCurrentBasket()
     const [isLoading, setIsLoading] = useState(false)
     const {mutateAsync: createOrder} = useShopperOrdersMutation('createOrder')
+    const {passwordless, social} = getConfig().app.login
```

Comment:
is there a reason we do `const {passwordless = {}, social = {}} = getConfig().app.login || {}` in use-auth-modal.js but just `const {passwordless, social} = getConfig().app.login` here?